upgrade/1.2/scripts/upgrade/prerequisites.sh

```diff
@@ -296,6 +296,16 @@ state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
 if [[ $state_recorded == "0" && $(hostname) == "ncn-m001" ]]; then
     echo "====> ${state_name} ..."
     {
+    set +e
+    nexus-cred-check () {
+        pod=$(kubectl get pods -n nexus --selector app=nexus -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' | grep -v nexus-init);
+        kubectl -n nexus exec -it $pod -c nexus -- curl -i -sfk -u "admin:${NEXUS_PASSWORD:=admin123}" -H "accept: application/json" -X GET http://nexus/service/rest/beta/security/user-sources >/dev/null 2>&1; 
```

Comment:
Should "admin:${NEXUS_PASSWORD:=admin123}" be
"admin:${NEXUS_PASSWORD:=$(kubectl get secret -n nexus nexus-admin-credential --template '{{.data.password}}' | base64 -d)}" ??

Reply:
Not in the 1.2 update. Unless I am incorrect this is for upgrading to 1.2 which means Nexus does not have a password other than default and this upgrade will give it a non-default password. So the default password prior to upgrading to 1.2 will be `admin123`.

Reply:
I'm also not sure if we can expose passwords in the scripts. Are there other cases where passwords are visible?

Reply:
We do have a precedent already for exposing the default nexus admin password. 
Examples:
nexus-setdefault-credential in lib/install.sh (github.hpe.com:hpe/hpc-shastarelm-release)
bin/nexus-curl-auth (github.com:Cray-HPE/nexus-setup.git)